### PR TITLE
blog: February newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Usual drill, @tbrisker / @orrabin / @rahulbajaj0509 review? :)

### CODE REVIEW

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+Welcome back to the Foreman Newsletter! We kinda slipped up in January, sorry
+about that, but we're back with 2 months of news, gossip, and releases for you!
+
+### Changes in the community team
+
+At the start of February, I made the difficult decision to step down as
+community manager. It's been an honour and a delight to help shape this
+community over the last few years, and to just be a part of it before that. You
+can read more about my (our?) history in my dedicated blog post [here][blog1]
```
Comment (evgeni) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]:
```suggestion
can read more about my (our?) history in [the dedicated blog post][blog1]
```

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+We can't, however, leave the community without a community lead, and so there
+has been some other changes to address this. I'm delighted to announce that Ori
+Rabin is [taking over as community lead][blog2] and Rahul Bajaj is joining the
+team, as well - read his introduction [here][blog3].
```
Comment (evgeni) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]:
```suggestion
team, as well - [read his introduction post][blog3].
```

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+I'm certain that along with Tomer, the new community team is ready to take
+Foreman on to bigger and better things!
+
+[blog1]: https://theforeman.org/2019/02/stepping-down.html
+[blog2]: https://theforeman.org/2019/02/introducing-the-new-community-team.html
+[blog3]: https://theforeman.org/2019/02/introducing-myself.html
+
+### Conference roundup
+
+As ever, Foreman was present at DevConf CZ, FOSDEM, and CfgMgmtCamp this year,
```
Comment (evgeni) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]:
```suggestion
As ever, Foreman was present at DevConf.CZ, FOSDEM, and CfgMgmtCamp this year,
```

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+and as usual had a hackday in Gent as well. This year was as busy as ever, with
+lots of traffic at the booths, lots of hats given out (wow, they were popular
+in the Brussels snowstorm...), and a great set of talks in our track at
```
Comment (evgeni) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: "we" also had talks in other tracks, is that smth worth mentioning?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+over 30 people attending. It's wonderful to see the community growing
+year-on-year - keep it going!
+
+### Upcoming Releases (RCs for 1.21 and Katello 3.11)
+
+Work on 1.21 has been progressing steadily while we've all been away having fun
+at conferences - credit to the release team for their dedication! RCs for 1.21
+are available, and we're actually up to RC5 already (although 1 & 2 were
+aborted, so really we're at 3), so it's time once again to ask you all to do
+some testing with us!
```
Comment (tbrisker) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: 1.21.0 will be GA tomorrow/thursday, so no point asking for testing help at this point - build is already in progress.

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md (hunk 4 above)
Comment (GregSutcliffe) [reply _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: True indeed, thanks for all the fixes, will sort that.

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md (hunk 5 above)
Comment (GregSutcliffe) [reply _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: Aha, good to know. Shall I hold off and wait for the release?

Comment (tbrisker) [reply _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: yes, that would be best.

**Comment by GregSutcliffe** [comment]
Updated with 1.21 release, squashed commits (thanks though, @evgeni :P)

File: _posts/2019-02-26-foreman-community-newsletter---february-2019.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+team, as well - [read his introduction post][blog3].
+
+I'm certain that along with Tomer, the new community team is ready to take
+Foreman on to bigger and better things!
+
+[blog1]: https://theforeman.org/2019/02/stepping-down.html
+[blog2]: https://theforeman.org/2019/02/introducing-the-new-community-team.html
+[blog3]: https://theforeman.org/2019/02/introducing-myself.html
+
+### 1.21 Released (RCs Katello 3.11 too)!
```
Comment (tbrisker) [inline _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: English not is, this sentence :yoda:

Comment (GregSutcliffe) [reply _posts/2019-02-26-foreman-community-newsletter---february-2019.md]: meh

**Comment by GregSutcliffe** [comment]
@tbrisker feel free to update the TODO url and push when 1.21 is live

**Comment by tbrisker** [comment]
gha force pushing to someone else's branch is hard

**Comment by tbrisker** [comment]
merged as fcde4bf9b, thanks @GregSutcliffe !

**Comment by GregSutcliffe** [comment]
Thanks @tbrisker !

**Pull request closed without merging**